tests/flit_cfg.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
I am not sure why we need this file. What features are implemented with this logic? Could we have a simpler mechanism?